ocs_ci/helpers/keyrotation_helper.py

```diff
@@ -278,6 +278,7 @@ def __init__(self):
             in [constants.VAULT_KMS_PROVIDER, constants.HPCS_KMS_PROVIDER]
         ):
             self.kms = get_kms_details()
+            self.kms.update_vault_env_vars()
```

Comment:
Maybe try this and see if it will help to resolve your issue.
```suggestion
            self.kms.gather_init_vault_conf()
            self.kms.update_vault_env_vars()
```